recipe/meta.yaml

```diff
@@ -1,19 +1,18 @@
 {% set name = "pmdarima" %}
-{% set version = "1.8.5" %}
+{% set version = "2.0.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 090a7e195e849059e6b7f9908d80539f63c0771d8f9c5da44472960f17fcc9fa
+  sha256: 91acb3f7c12f3cfd6263efe323d3b8f94b58dcd242d68f11560a6c717001b01f
 
 build:
   number: 0
-  # Unsatisfiable dependencies on win32 with python 3.10
-  skip: true  # [py<37 or (win32 and py>=310)]
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<37]
```

Comment:
It looks like the dependency that was missing for Python 3.10 on Windows has been added. So I removed that condition.